src/_scss/layouts/default/header/_warning.scss

```diff
@@ -36,12 +34,12 @@
             margin-left: 1rem;
             padding-bottom: rem(8);
```

Comment:
This padding isn't in the mock, and it's making the space below the text 24px instead of 16px.
The mock shows 16px above and below the text.